### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by NicolasMatekalo**
Implement MapFragment that leads to MapActivity.
This displays a Google Map which type can be changed.
Track user position.
Add tests for it.

### CODE REVIEW

File: app/src/main/res/layout/fragment_map.xml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".fragment.navigation.MapFragment">
+    tools:context=".fragment.navigation.CourseTabFragment">
```
Comment (mbouilloud) [inline app/src/main/res/layout/fragment_map.xml:6]: Why do you have "CourseTabFragment" here ? Shouldn't it be "MapFragment" ?

**Review by jmettler27 — APPROVED** [review]
Nothing to say, good job

File: app/src/main/res/layout/fragment_map.xml (hunk 1 above)
Comment (NicolasMatekalo) [reply app/src/main/res/layout/fragment_map.xml:6]: Yes indeed, a copy-pasta has gone wrong ^^

**Review by mbouilloud — APPROVED** [review]

**Comment by mbouilloud** [comment]
Sadly we cannot test the map :(

**Pull request merged**